scopes/lanes/ui/overview/empty-lane-overview.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { EmptyComponentGallery, EmptyComponentGalleryProps } from '@teambit/ui-foundation.ui.empty-component-gallery';
+import styles from './empty-lane-overview.module.scss';
+
+export type EmptyLaneProps = { name: string } & EmptyComponentGalleryProps;
+
+/**
+ * A component to show when the scope is empty
+ */
+export function EmptyLane({ name }: EmptyLaneProps) {
+  return (
+    <EmptyComponentGallery name={name}>
```

Comment:
forward props.. they even include name!

```suggestion
export function EmptyLane(props: EmptyLaneProps) {
  return (
    <EmptyComponentGallery {...props}>
```